sdk/formrecognizer/ai-form-recognizer/test/public/training.spec.ts

```diff
@@ -241,11 +240,10 @@ matrix(
       // #endregion
 
       it(`compose model (${buildMode})`, async function () {
-        const client = new DocumentModelAdministrationClient(
-          endpoint(),
-          makeCredential(useAad),
-          recorder.configureClientOptions({})
-        );
+        const client = new DocumentModelAdministrationClient(endpoint(), makeCredential(useAad), {
+          ...recorder.configureClientOptions({}),
+          audience: getAudience(),
+        });
```

Comment:
Is this the same? Avoids the object spread.

```suggestion
        const client = new DocumentModelAdministrationClient(endpoint(), makeCredential(useAad), 
          recorder.configureClientOptions({ audience: getAudience() }),
        );
```